alerting/src/main/kotlin/org/opensearch/alerting/AlertingPlugin.kt

```diff
@@ -240,6 +241,7 @@ internal class AlertingPlugin : PainlessExtension, ActionPlugin, ScriptPlugin, R
             ClusterMetricsInput.XCONTENT_REGISTRY,
             DocumentLevelTrigger.XCONTENT_REGISTRY,
             ChainedAlertTrigger.XCONTENT_REGISTRY,
+            RemoteMonitorTrigger.XCONTENT_REGISTRY,
```

Comment:
do we also need RemoteMonitorInput xcontent registry?

if yes, can you add appropriate tests to serialize

Reply:
we dont need `RemoteMonitorInput` because in `common-utils` `Input.parse` doesn't use it. https://github.com/opensearch-project/common-utils/blob/main/src/main/kotlin/org/opensearch/commons/alerting/model/Input.kt#L30
however, in `Trigger.parse` it is used. https://github.com/opensearch-project/common-utils/blob/main/src/main/kotlin/org/opensearch/commons/alerting/model/Trigger.kt#L41